pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -282,6 +282,9 @@ const (
 	// LabelWorkerPool is a deprecated constant for a label that indicates the worker pool the node belongs to
 	LabelWorkerPoolDeprecated = "worker.garden.sapcloud.io/group"
 
+	// LabelWorkerPoolSystemComponentsAllow is a constant that indicates whether the worker pool should host system components
+	LabelWorkerPoolSystemComponentsAllow = "worker.gardener.cloud/system-components"
```

Comment:
```suggestion
	LabelWorkerPoolSystemComponents = "worker.gardener.cloud/system-components"
```
Please also adjust doc string.